### ISSUE TITLE
SyntaxError: Unexpected token import when import maually

### ISSUE THREAD

**Issue by amazingSurge** [webpro/release-it#134]
When i import the release-it via code below
```
import releaseIt from 'release-it';
```

I got a error.

```
/node_modules/release-it/lib/index.js:1
(function (exports, require, module, __filename, __dirname) { import { version, help } from './cli';
                                                              ^^^^^^

SyntaxError: Unexpected token import
    at createScript (vm.js:74:10)
    at Object.runInThisContext (vm.js:116:10)
    at Module._compile (module.js:533:28)
    at Module._extensions..js (module.js:580:10)
    at Object.require.extensions.(anonymous function) [as .js] (/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/tasks/release.js:3:1)
    at Module._compile (module.js:569:30)
    at loader (/node_modules/babel-register/lib/node.js:144:5)
    at Object.require.extensions.(anonymous function) [as .js] (/node_modules/babel-register/lib/node.js:154:7)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
```

Any help?

**Comment by webpro** [comment]
Hi, @amazingSurge!

First of all, release-it is a CLI tool. You can install and run it from a terminal:

```
$ npm install -g release-it
$ release-it 
```

You could import it in your own Node script, but the code would need to be transpiled to syntax that the version of Node.js it runs in is supported. When running from the command line, this happens in [bin/release-it.js](https://github.com/webpro/release-it/blob/master/bin/release-it.js). A minimal example using Babel looks like this:

```
require('babel-register')
const releaseIt = require('release-it').default;
releaseIt();
```

Note that this also requires a Babel configuration. This is the [.babelrc](https://github.com/webpro/release-it/blob/master/.babelrc) that release-it is using. As you are using ESM syntax (`import`) I assume you are already using some kind of transpilation.

Unfortunately, it is currently unsupported to pass `releaseIt()` a configuration object.

But my question to you: what is your use case to run release-it from your own script?

**Comment by amazingSurge** [comment]
HI @webpro 
Thanks for the quick reply.
I am using with gulp.

SEE https://github.com/amazingSurge/web-starter-kit/blob/master/tasks/release.js

It's uncomment currently.

We had https://github.com/amazingSurge/web-starter-kit/blob/master/.babelrc

It seems release-it don't working well with env preset.

In https://github.com/amazingSurge/web-starter-kit/blob/9bab35b0f8ec7afea9d986845e9a4e06db055947/.babelrc

It seems works fine.

Best Regards

**Comment by webpro** [comment]
Why don't you use an npm script, like so:

```
"scripts": {
  "release": "release-it"
}
```

* You'll need to rename `.release.json` to `.release-it.json`. Please see the [release notes for v3](https://github.com/webpro/release-it/releases/tag/3.0.0).
* Some more ideas, looking at the `package.json`:
  * In the release-it config you could use `"buildCommand": "gulp build"`
  * Maybe also `"src.afterReleaseCommand": "gulp deploy --production"`

**Comment by webpro** [comment]
I've released [5.0.0-beta.0](https://github.com/webpro/release-it/releases/tag/5.0.0-beta.0):

* Can be imported in Node v8 without transpilation (#157)
* Allows to pass options to main function (#151)

This may help in your use case. Feel free to discuss further.

**Comment by amazingSurge** [comment]
Thank you!
